.github/workflows/test.yaml

```diff
@@ -16,5 +16,9 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: Swatinem/rust-cache@v1
-      - run: cargo test
+      - uses: foundry-rs/foundry-toolchain@v1
+      - run: anvil --fork-url 'https://eth.merkle.io' --fork-block-number 20920411 &
```

Comment:
This would require all tests to work on the same for block.
But I guess fixing this is not worth it overall if we don't regularly add more tests.